README.adoc

```diff
@@ -119,6 +119,20 @@ $ idea .
 $ open servicetalk.ipr
 ----
 
+.To run the test case:
+[source,shell]
+----
+./gradlew clean test
```

Comment:
```suggestion
./gradlew test
```

Reply:
@shekhar-rajak I think you wanted to add this to the README because you did verify your changes on a PR with `gradlew build` but still there were some build errors that were uncovered by the PRB. Can you elaborate what areas `build` task did not cover?

Reply:
I think I didn't check the linting error.  This change will help when someone only wants to run test or quality. WDYT ?

Reply:
Ok I would then agree with @idelpivnitskiy 's suggestion to modify language and not list specific tasks.

Reply:
Thanks! I have done the changes accordingly.

Reply:
There is almost no difference between running `test quality` instead of `build` task, performance and execution time will be the same. We run them separately on to avoid running `quality` task twice: for JDK8 and JDK11. This is the only reason.
If `build` misses something compare to `test quality` this is an issue that needs to be addressed.